### PR TITLE
LPS-202007 buildLang

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useConditionValue.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		fetcher: () => RulesService.getRoles(),
+		key: [CACHE_KEYS.roles],
+	});
+
+	const users = useCache({
+		fetcher: () => RulesService.getUsers(),
+		key: [CACHE_KEYS.users],
+	});
+
+	const segments = Object.values(config.availableSegmentsEntries);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useConditionValue.ts:28]: If you don't do the Object.values here you can simply access the segments with `config.availableSegmentEntries[id]` right?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js [hunk 2]
```diff
@@ -120,6 +129,111 @@ function Rule({onDelete, onEdit, rule}) {
...
+	const {label: actionLabel} = ACTION_ITEMS.find(
+		({value}) => value === action.action
+	);
+
+	const item = useSelector((state) => state.layoutData.items[action.itemId]);
+
+	const name = useSelectorCallback(
+		(state) => selectLayoutDataItemLabel(state, item),
+		[item]
+	);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js:235]: You can merge this two into one useSelectorCallback call right?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js [hunk 3]
```diff
@@ -120,6 +129,111 @@ function Rule({onDelete, onEdit, rule}) {
...
+	const valueLabel = useConditionValue({
+		id: condition.value,
+		type: condition.condition,
+	});
+
+	return (
+		<>
+			<span
+				className={classNames('font-weight-semi-bold', {
+					'text-uppercase': Boolean(index),
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js:202]: This is a bit weird, should be more readable something like index !== 0 ? or index > 0 ?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useConditionValue.ts (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useConditionValue.ts:28]: Yep, much more beautiful, thanks

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js:235]: Actually yes

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js (hunk 3 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_rules/components/RulesList.js:202]: Yeah, changing it!

**Comment by victorg1991** [comment]
Niiice :clap:

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:reevaluate:1317401_9988

**Comment by ealonso** [comment]
ci:test:stable

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/144246

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**